keyboards/puck/config.h

```diff
@@ -0,0 +1,34 @@
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x6060
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    OKKeebs, LLC
```

Comment:
Commas cause issue here, to the point of stopping it from compiling.

If you could replace it with: 
```suggestion
#define MANUFACTURER    OKKeebs LLC
```